StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -190,22 +190,20 @@ final class PaymentSheetLoader {
         return intent
     }
 
-    static func additionalParams(configuration: PaymentSheet.Configuration) -> [String: Any]? {
-        if let ephemeralKey = configuration.customer?.ephemeralKeySecret {
-            return ["legacy_customer_ephemeral_key": ephemeralKey]
+    static func customerEphemeralKey(configuration: PaymentSheet.Configuration) -> String? {
+        guard let ephemeralKey = configuration.customer?.ephemeralKeySecret else {
+            return nil
         }
-        return nil
+        return ephemeralKey
     }
```

Comment:
This seems unnecessary as a method, I think its caller can simply do `configuration.customer?.ephemeralKeySecret`